fast_llm/tensor.py

```diff
@@ -312,15 +312,15 @@ def init_(meta: ParameterMeta, tensor: torch.Tensor, generator: torch.Generator)
 init_ones_ = init_fill_(1.0)
 
 
-def init_normal_(mean=0.0, std=1.0):
+def init_normal_(mean=0.0, std=1.0, min_val=None, max_val=None):
     def init_(meta: ParameterMeta, tensor: torch.Tensor, generator: torch.Generator):  # noqa
-        return tensor.normal_(mean, std, generator=generator)
+        return tensor.clamp(tensor.normal_(mean, std, generator=generator), min=min_val, max=max_val)
```

Comment:
I don't think that works. Probably `tensor.normal_.clamp_`?
Also let's put in a `if` so there is no clamp call by default.